src/pages/KyberDAO/KNCUtility/Table.tsx

```diff
@@ -0,0 +1,55 @@
+import { rgba } from 'polished'
+import styled from 'styled-components'
+
+export const TableWrapper = styled.div`
```

Comment:
duplicate style with TxTable?

Reply:
Yes. I clone them into 2 copy, each has some different design from each other. It would be better to keep them separately like current.